src/index.js

```diff
@@ -129,7 +129,7 @@ const propertiesList = (definition: Object) => {
 };
 
 const withExact = (property: string): string => {
-  const result = property.replace(/{[^|]/g, "{|").replace(/[^|]}/g, "|}");
+  const result = property.replace(/{[^|}]/g, "{|").replace(/[^|{]}/g, "|}");
```

Comment:
I feel like this should be a lookahead rather than a match - hesitant to change as this might be deliberate.